examples/todo-list/src/controllers/todo-list.controller.ts

```diff
@@ -38,8 +38,17 @@ export class TodoListController {
       },
     },
   })
-  async create(@requestBody() obj: TodoList): Promise<TodoList> {
-    return await this.todoListRepository.create(obj);
+  async create(
+    @requestBody({
+      content: {
+        'application/json': {
+          schema: getModelSchemaRef(TodoList, {exclude: ['id']}),
+        },
+      },
+    })
+    todoList: Omit<TodoList, 'id'>,
```

Comment:
Should we support `include` for `todoList: Pick<TodoList, 'id'>`?

Reply:
I'm fine to do it in a separate PR.

Reply:
I am not sure what would you use `include` for? A kind of a white-list? That would make sense :+1:

Let's discuss `include` and `Pick` in a separate PR. I think it's best to wait until there is (user) demand for that feature.